src/test/scala/data.scala

```diff
@@ -4,6 +4,8 @@ import ohnosequences.db.rnacentral
 import ohnosequences.fastarious.fasta._
 import rnacentral.{EntryAnnotation, RNACentralData, RNAID, Version, iterators}
 import java.io.File
+import ohnosequences.s3.{Error => S3Error}
+import rnacentral.s3Helpers.getCheckedFileIfDifferent
 
 object data {
 
```

Comment:
Just a suggestion: maybe `cleanLocalFolder` could be erased from here (it is not being used anywhere else). Apart from that we know have an specific function in `files` to delete directories that outputs `File.Error` if something went wrong.

Reply:
Oops, legacy code I forgot to delete, thanks!